cognite/v06/sequences.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""Sequences Module
+
+This module mirrors the Sequences API.
+
+https://doc.cognitedata.com/api/0.6/#tag/Sequences
+"""
+import json
+from typing import List
+
+from cognite import _utils, config
+from cognite.v06.dto import Sequence, Column
+
+
+def post_sequences(
+        sequences: List[Sequence],
+        **kwargs
+):
+    """Create a new time series.
+
+    Args:
+        sequences (list[v06.dto.Sequence]): List of sequence data transfer objects to create.
+
+    Keyword Args:
+        api_key (str): Your api-key.
+        project (str): Project name.
+
+    Returns:
+        The created sequence
+    """
+
+    api_key, project = config.get_config_variables(kwargs.get("api_key"), kwargs.get("project"))
+    url = config.get_base_url() + "/api/0.6/projects/{}/sequences".format(project)
+
+    body = {"items": [sequence.__dict__ for sequence in sequences]}
```

Comment:
This post body will be using snake_case not camelCase, and will therefore yield 400 always.

Reply:
The test passes, and I can confirm that the sequence was actually created. So no 400 there

(am I missing something obvious now?)